shell/edit/management.cattle.io.project.vue

```diff
@@ -134,7 +135,7 @@ export default {
         saveCb(true);
         this.$router.replace(this.value.listLocation);
       } catch (ex) {
-        this.errors.push(ex);
+        this.errors = uniq([...this.errors, ex.message]);
```

Comment:
We might want to do something different. I can't validate this atm, but in theory users would not see new errors if a previous one exists?

For example
1. user receives "error 1"
2. user changes form, receives "error 2"
3. user reverts change, user does not see error with latest attempt as it appears at start of stack before "error 2"

I think the pattern in the norm cru save is to just clear the error array on save. Would there be any issues doing it here?

Reply:
> I think the pattern in the norm cru save is to just clear the error array on save. Would there be any issues doing it here?

Yeah, I think that makes more sense. We'll never clear out irrelevant errors otherwise.

Reply:
The only perceptible "downside" to this approach is that we will only display one server error at a time. I think that this is more correct than either of the previous approaches.

Reply:
Maybe a bit out of the scope, but ideally we should run some validation against these values before sending the request.

Reply:
Got my reference slightly wrong. It's not cru resource where we do the save, it's the create-edit-view mixin - `shell/mixins/create-edit-view/impl.js` `save` 
```
      if ( this.errors ) {
        clear(this.errors);
      }
```